site/docs/components/overview.mdx

```diff
@@ -27,6 +27,23 @@ We strive to achieve the following quality attributes on all of our components:
 
 The status of these can be found on the documentation page for each component, or on the [Component health](/component-health) page, which shows the status of all components.
 
+### API
+- **Avoid language overloading**: Avoid prop names that are used in HTML, such as `type`.
+- **Lean on existing mental models**: Things like input `onChange` should not stray from the default behavior that React has defined.
+- **Encapsulation:** Components should only be concerned about their own form and function and not about how they appear in relation to other components.
+- **Favor composition**: This allows us to have a more succinct API that encourages code reuse.
+- **RTL support**: Avoid using left/right props or values, instead use start/end and support RTL automatically.
```

Comment:
```suggestion
- **RTL support**: Avoid using left/right props or values, instead use start/end and support RTL automatically. See [CSS Logical Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) for more information.
```

Reply:
Some folks might not have come across logical properties just yet :)

Reply:
🤔 this isn't necessarily tied to that. Most people are familiar with start/end over left/right with flexbox, and we don't actually have to use logical properties under the hood to support start/end either / we can't because of legacy Edge.

Reply:
Ah! My mistake - ignore this.